Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftWindowsEventFirewall.yaml

```diff
@@ -166,7 +97,7 @@ ParserQuery: |
         | project-away EventData
     };
   // Main query -> outputs both schemas as one normalized table
-  union WindowsFirewall_SecurityEvent, WindowsFirewall_WindowsEvent 
+  WindowsFirewall_WindowsEvent 
```

Comment:
I don't think we need to split, since we just now have just one schema

Reply:
done